src/__test__/_components/OfficeSectionsContainer.test.js

```diff
@@ -0,0 +1,24 @@
+import expect from 'expect';
+import { mapStateToProps } from '../../_components/OfficeSections/OfficeSectionsContainer';
+
+describe('mapStateToProps', () => {
+  it('returns the expected props', () => {
+    const state = {
+      officeSections: {
```

Comment:
Since this is name officeSections, the nested props should be `list`, `count` instead of prefixing it again. Do same for the expected section and all other props like that